rotkehlchen/types.py

```diff
@@ -523,3 +510,30 @@ def deserialize(cls: Type['AddressbookEntry'], data: Dict[str, Any]) -> 'Address
 class AddressbookType(SerializableEnumMixin):
     GLOBAL = 1
     PRIVATE = 2
+
+
+class DataImportSource(SerializableEnumMixin):
+    COINTRACKING = 1
+    CRYPTOCOM = 2
+    BLOCKFI_TRANSACTIONS = 3
+    BLOCKFI_TRADES = 4
+    NEXO = 5
+    GITCOIN = 6
+    SHAPESHIFT_TRADES = 7
+    UPHOLD_TRANSACTIONS = 8
+    BISQ_TRADES = 9
+    BINANCE = 10
+
+    @classmethod
+    def deserialize(cls: Type['DataImportSource'], value: str) -> 'DataImportSource':
```

Comment:
Added only `deserialize` method since `DataImportSource` is not supposed to be used in API responses as a json key.